### DISCUSSION TITLE
Have you considered adding musixml's mxl compressed format loading?

### DISCUSSION THREAD

**Discussion by DMC-HYX** [CoderLine/alphaTab#1435] [Ideas]
Have you considered adding musixml's mxl compressed format loading?

**Comment by Danielku15** [comment]
Until now it wasn't directly on [the radar](https://github.com/orgs/CoderLine/projects/12), but I added an item to the roadmap: https://github.com/orgs/CoderLine/projects/12?pane=issue&itemId=59641230

**Reply by Danielku15** [reply]
@DMC-HYX Feature is implemented as part of https://github.com/CoderLine/alphaTab/issues/1483 and will be available in the next 1.4 nightly build.
